modules/developer_manual/pages/core/ocs-capabilities.adoc

```diff
@@ -60,10 +60,7 @@ Stored under the `files` capabilities element, this returns the server's support
 |File versioning
 |`versioning`
 
-|Navigating directly to a file's version, comments, and sharing pane
-|`privatelLinks` and `privateLinksDetailsParam`
```

Comment:
I took these lines out completely, because I am not sure anyway what "Navigating directly" has to do with `privateLinks` - I will look into it...

Reply:
`privateLinks` capability was added in core PR https://github.com/owncloud/core/pull/29021
It refers to the capability that core DAV can return a private link to a file.
So I do not see what this has to do with "Navigating directly"